pkg/idtools/parser.go

```diff
@@ -33,9 +26,12 @@ func parseTriple(spec []string) (container, host, size uint32, err error) {
 
 // ParseIDMap parses idmap triples from string.
 func ParseIDMap(mapSpec []string, mapSetting string) (idmap []IDMap, err error) {
-	stdErr := fmt.Errorf("error initializing ID mappings: %s setting is malformed", mapSetting)
+	stdErr := fmt.Errorf("error initializing ID mappings: %s setting is malformed: %q", mapSetting, mapSpec)
```

Comment:
I think the error should include the syntax. If I misconfigured it, I may not know what it should look like.